Source/Digst.OioIdws.Wsp/BasicPrivilegeProfile/BasicPrivilegeProfileSchema_v102.xsd

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
XSD filerne bruges ikke, men jeg inkluderede den for dokumentationens skyld. Jeg er usikker på om det ønskes, eller om vi blot skal linke til specifikationen.